src/Command/CreateCommand.php

```diff
@@ -105,6 +105,7 @@ public function configure(): void
             ->addOption('table-comment', null, InputOption::VALUE_OPTIONAL, 'Table comment')
             ->addOption('and', null, InputOption::VALUE_OPTIONAL, 'And junction')
             ->addOption('namespace', null, InputOption::VALUE_OPTIONAL, 'Namespace migration')
+            ->addOption('transactional', 't', InputOption::VALUE_OPTIONAL, 'Transactional migration', false)
```

Comment:
```suggestion
            ->addOption('transactional', 't', InputOption::VALUE_NONE, 'Transactional migration')
```

With this code `$input->getOption('transactional')` returns boolean value.